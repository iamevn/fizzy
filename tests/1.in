2,4:hello
3:world
13
