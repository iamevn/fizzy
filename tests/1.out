1
hello
world
hello
5
helloworld
7
hello
world
hello
11
helloworld
13
